### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ESaracay**
Screenshot 2020-08-31 at 7 15 36 PM
Screenshot 2020-08-31 at 7 15 09 PM
This PR adds loading spinners to all components that make servlet calls and utf-8 support. It also includes a product demo video in the home page.

### CODE REVIEW

File: frontend/src/homePage/LandingPage.js [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
+      this.message = "My Folders";
+      this.color = "#F4B400";
+      this.clickHandling = this.props.folderClick;
     }
   }
 
+  render() {
+    return (
+      <div className='LandingPage'>
+        <h1>Frame.cards learning made easy</h1>
```
Comment (zmr) [inline frontend/src/homePage/LandingPage.js]: nit: Frame.cards: Learning Made Easy

or Frame.cards - Learning Made Easy

are you typically capitalizing cards or not?

File: frontend/src/main-components/CreateCardContent.js [hunk 2]
```diff
@@ -61,7 +65,7 @@ class CreateCardContent extends Component {
           if (prevState.toLang != toLang || prevState.text != text) {
             return;
           }
```
Comment (zmr) [inline frontend/src/main-components/CreateCardContent.js:72]: any reason not to setState loading=false right after getTranslation returns?

Comment (ESaracay) [reply frontend/src/main-components/CreateCardContent.js:72]: This means that the user typed in a different word and the request is still being processed.

File: frontend/src/homePage/About.css [hunk 3]
```diff
@@ -1,4 +1,5 @@
 .About{
```
Comment (tylerhou) [inline frontend/src/homePage/About.css]: Formatting and naming.

File: frontend/src/homePage/LandingPage.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .LandingPage{
```
Comment (tylerhou) [inline frontend/src/homePage/LandingPage.css]: Formatting and naming.

File: frontend/src/main-components/CreateCardContent.js [hunk 5]
```diff
@@ -18,6 +20,7 @@ class CreateCardContent extends Component {
       folder: "",
       uploadUrlFetched: false,
       imageUploadUrl: "",
+      loading:false,
```
Comment (tylerhou) [inline frontend/src/main-components/CreateCardContent.js]: Formatting.

File: frontend/src/main-components/CreateCardContent.js (hunk 2 above)
Comment (tylerhou) [reply frontend/src/main-components/CreateCardContent.js:72]: Add comment?

File: frontend/src/main-components/CreateCardContent.js [hunk 6]
```diff
@@ -93,7 +97,11 @@ class CreateCardContent extends Component {
 
   render() {
     if (!this.state.uploadUrlFetched) {
-      return <h1>Loading</h1>;
+      return (
+        <div className='loadingContainer'>
+        <PageLoading></PageLoading>
```
Comment (tylerhou) [inline frontend/src/main-components/CreateCardContent.js]: Formatting.

**Review by tylerhou — CHANGES_REQUESTED** [review]

File: frontend/src/homePage/LandingPage.js (hunk 1 above)
Comment (ESaracay) [reply frontend/src/homePage/LandingPage.js]: Yeah it should be capatialized.

File: frontend/src/main-components/CreateCardContent.js (hunk 2 above)
Comment (ESaracay) [reply frontend/src/main-components/CreateCardContent.js:72]: There is a comment on line 60 but it's cut off on the preview for some reason.

File: frontend/src/homePage/About.css (hunk 3 above)
Comment (ESaracay) [reply frontend/src/homePage/About.css]: done.

File: frontend/src/homePage/LandingPage.css (hunk 4 above)
Comment (ESaracay) [reply frontend/src/homePage/LandingPage.css]: done.

File: frontend/src/main-components/CreateCardContent.js (hunk 5 above)
Comment (ESaracay) [reply frontend/src/main-components/CreateCardContent.js]: done.

File: frontend/src/main-components/CreateCardContent.js (hunk 6 above)
Comment (ESaracay) [reply frontend/src/main-components/CreateCardContent.js]: done.

File: frontend/src/homePage/LoginButton.css [hunk 7]
```diff
@@ -8,7 +8,7 @@
     border-radius: 25px;
   }
 
-.MyButton a{
+.myButton a{
```
Comment (tylerhou) [inline frontend/src/homePage/LoginButton.css]: Space after a

File: frontend/src/sub-components/TranslationLoading.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+class TranslationLoading extends React.Component {
+  render() {
+    if (this.props.loading) {
+      return (
+        <div>
+          <PulseLoader size={7} margin={3} color={"#136F9F"} loading={true} />
+        </div>
+      );
+    } 
+      return <div></div>;
```
Comment (tylerhou) [inline frontend/src/sub-components/TranslationLoading.js]: Formatting

File: src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java [hunk 9]
```diff
@@ -7,6 +7,7 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.lang.*;
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java]: No wildcard imports (see Google Java style guide)

**Review by tylerhou — APPROVED** [review]

File: frontend/src/homePage/LoginButton.css (hunk 7 above)
Comment (ESaracay) [reply frontend/src/homePage/LoginButton.css]: done.

File: frontend/src/sub-components/TranslationLoading.js (hunk 8 above)
Comment (ESaracay) [reply frontend/src/sub-components/TranslationLoading.js]: done.

File: src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java (hunk 9 above)
Comment (ESaracay) [reply src/main/java/com/google/sps/servlets/UploadOnBlobstoreServlet.java]: done.

**Pull request merged**